### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by HeroicEric**

### CODE REVIEW

File: app/views/cars/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+               _--`-___                            ___-'--_
+             /~     `\ ~~~~~~~~------------~~~~~~~~ /'     ~\
+            /|        `\         ________         /'        |\
+           | `\   ______`\_      \------/      _/'______   /' |
+           |   `\_~-_____\ ~-________________-~ /_____-~_/'   |
+           `.     ~-__________________________________-~     .'
+            `.      [_______/------|~~|------\_______]      .'
+             `\--___((____)(________\/________)(____))___--/'
+              |>>>>>>||                            ||<<<<<<|
+            `\<<<<</'                            `\>>>>>/'                             `\>>>>>/'
```
Comment (HeroicEric) [inline app/views/cars/index.html.erb:22]: lol

File: app/views/cars/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+       __.--""     __        ,'                   o \           __        [__|
+  __-""=======.--""  ""--.=================================.--""  ""--.=======:
+ ]       [w] : /        \ : |========================|    : /        \ :  [w] :
+ V___________:|          |: |========================|    :|          |:   _-"
+  V__________: \        / :_|=======================/_____: \        / :__-"
+  -----------'  ""____""  `-------------------------------'  ""____""
+</pre>
+
+<% @cars.each do |car| %>
+  <ul class = list>
```
Comment (HeroicEric) [inline app/views/cars/index.html.erb:39]: `<ul class="list">`

File: app/views/manufacturers/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+         8                        NNNNNNNN         8                 dNNNNNN
+   ______8__........----------""""""""""""------...8..______         NNNNNNN
+  _________........----------""""""""""""------......_____  """""----NNNNNNN
+                                                           """""----....___ """--
+                                                                           """---
+  </pre>
+</div>
+
+<% @manufacturers.each do |manufacturer| %>
+  <ul class = list>
```
Comment (HeroicEric) [inline app/views/manufacturers/index.html.erb:49]: same as above

**Comment by HeroicEric** [comment]
Great job! You can close whenever you want.

**Comment by RSid** [comment]
Thanks! And d'oh, quotes.

**Pull request merged**